openreview/journal/journal.py

```diff
@@ -403,6 +403,12 @@ def enable_blocked_authors(self):
 
     def get_blocked_authors_id(self):
         return f'{self.venue_id}/Submission_Banned_Users'
+    
+    def has_journal_to_conference_certification(self):
+        return self.settings.get('journal_to_conference_certification', False)
```

Comment:
what does it mean `journal_to_conference_certification`?

Reply:
this is just the name of this specific certification they want to add --> J2C Certification